docs/reference/versions.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Versions"
+layout: default
+sort: 8
+---
+
+# Versions and deprecation
+{: .no_toc}
+
+---
+
+## Supported versions
+
+Node Feature Discovery follows [semantic versioning](https://semver.org/) where
+the version number consists of three components, i.e. **MAJOR.MINOR.PATCH**.
+
+The most recent two minor releases (or release branches) of Node Feature
+Discovery are supported. That is, with X being the latest release, **X** and **X-1**
+are supported and **X-1** reaches end-of-life when **X+1** is released.
+
+## Deprecation policy
```

Comment:
will you call out any deprecation in release notes ?

e.g another section in release notes (after the "Whats Chaged") section
or in the same section but the PR title be in a specific format like: `[DEPRECATE] remove feature FOO` so that it "pops out" in release notes ?

Reply:
> will you call out any deprecation in release notes ?
> 
> e.g another section in release notes (after the "Whats Chaged") section or in the same section but the PR title be in a specific format like: `[DEPRECATE] mark feature FOO as deprecated` so that it "pops out" in release notes ?

Yes, definitely. We've already had a "Deprecations" section under Changelog/What's new in the past few releases (e.g. [v0.13.0](https://github.com/kubernetes-sigs/node-feature-discovery/releases/tag/v0.13.0). Maybe deprecations should/could be a separate main section to spot that more easily, dunno 🧐 We can discuss this when writing release notes for [v0.14.0](https://github.com/kubernetes-sigs/node-feature-discovery/issues/1250) (or you can comment it there already if you have opinions on that).

Good suggestion about the PR titles. We could add some style-guide/best-practices/contrib-guidelines e.g. in `docs/contributing/` or somewhere. Feel free to help with that 😉